integration/README.md

```diff
@@ -0,0 +1,60 @@
+# Integration Tests
+
+This tests core use cases for webboot such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
+
+     or 
+
+    UROOT_QEMU=/usr/bin/qemu-system-x86_64 UROOT_KERNEL=$HOME/linux/arch/x86_64/boot/bzImage go test
+
+When the QEMU arch is not amd64, set the `UROOT_TESTARCH` variable. For
+example:
+    UROOT_TESTARCH=arm go test
+
+Currently, only amd64 and arm are supported.
+
+## Steps to get the kernel (bzImage)
+#Step 1: clone the open source linux directory:
+git clone https://github.com/torvalds/linux.git
+cd linux 
+
+#Step 2: checkout the latest Linux version 
+git checkout v4.17
+
+#Step 3: Download a config file from u-root and replace the current .config in linux with u-root .config file
+wget https://raw.githubusercontent.com/u-root/u-root/master/.circleci/images/test-image-amd64/config_linux4.17_x86_64.txt
+
+
+#Step 4: build the kernel 
+Make -j12 
+
+#Step 5: edit RUNLOCAL binary inside your integration folder to add a path to the kernel you built to as follows: 
```

Comment:
no, they should not need to edit this file at all. All they need to do is set those two variables and run the file.

Reply:
in fact once they build the kernel they can use the command line from line 17, right?

Reply:
good point.

Reply:
We realized one should not have to edit this to simply run it locally. Instead, yes, you are right, they can just use the command line from 17 to run locally.

Reply:
if they want to use the RUNLOCAL file, they can just drop a bzImage and just run "./RUNLOCAL" without editing this file, this can also work and it will run the integration test in that way as well.